utils/node_local_test.py

```diff
@@ -1191,6 +1193,37 @@ def import_daos(server, conf):
     daos = __import__('pydaos')
     return daos
 
+class daos_cmd_return():
+    """Class to enable pretty printing of daos output"""
+
+    def __init__(self):
+        self.rc = None
+        self.valgrind = []
+        self.cmd = []
+
+    def __getattr__(self, item):
+        return getattr(self.rc, item)
+
+    def __str__(self):
+        if not self.rc:
+            return 'daos_command_return, process not yet run'
+        output = "CompletedDaosCommand(cmd='{}')".format(' '.join(self.cmd))
+        output += '\nReturncode is {}'.format(self.rc.returncode)
+        if self.valgrind:
+            output += "\nProcess ran under valgrind with '{}'".format(' '.join(self.valgrind))
+
+        try:
+            pp = pprint.PrettyPrinter()
+            output += '\njson output:\n' + pp.pformat(self.rc.json)
+        except AttributeError:
+            for line in self.rc.stdout.splitlines():
+                output += '\nstdout: {}'.format(line)
+
+        if self.rc.stderr != b'':
+            for line in self.rc.stderr.splitlines():
+                output += '\nstderr: {}'.format(line)
```

Comment:
Optional: I don't think the `if` is needed, unless it's possible `stderr` doesn't exist at all. E.g.
```
>>> b'non-empty'.splitlines()
[b'non-empty']
>>> b''.splitlines()
[]
```
```suggestion
        for line in self.rc.stderr.splitlines():
            output += '\nstderr: {}'.format(line)
```